samples/subsys/settings/sample.yaml

```diff
@@ -0,0 +1,33 @@
+sample:
+  name: Settings Sample
+
+tests:
+  sample.subsys.settings:
+    tags: settings
+    depends_on: nvs
```

Comment:
Remove that please, this limits building/testing to qemu_x86 which is the only platform that has nvs set. You either limit by dependency or by whitelisting, not both. Right now the test fails on native_posix.

Reply:
Maybe better to remove this dependency? Sample is able to run using NVS and FS as bak-ends.

Reply:
@nashif I removed this line `depends_on: nvs`

Reply:
I see - the issue is caused by different key read-out order in for FS back-end - its expected behavior, need to fix the harness.

Reply:
fixed